charmcraft.yaml

```diff
@@ -0,0 +1,20 @@
+type: "charm"
+bases:
+  - build-on:
+      - name: "ubuntu"
+        channel: "20.04"
```

Comment:
Can we use 22.04 here?

Reply:
If this is the case, it would be great to also put 22.04 in all the CI files.

Reply:
✅